src/Integration.UnitTests/LocalServices/ErrorListInfoBarController.QualityProfileBackgroundProcessorTests.cs

```diff
@@ -182,7 +152,40 @@ public void QualityProfileBackgroundProcessor_QueueCheckIfUpdateIsRequired_NoPro
         }
 
         [TestMethod]
-        public void QualityProfileBackgroundProcessor_BackgroundTask_DifferentTimestamp_RequiresUpdate()
+        public void QualityProfileBackgroundProcessor_BackgroundTask_DifferentTimestamp_RequiresUpdate_Connected()
```

Comment:
Testing:
I've duplicated all of the legacy mode tests in _SolutionBindingInformationProviderTests_ so that they are also run for connected mode.

The _SolutionBindingInformationProvider_ is used by the higher-level _ErrorListInfoBarController_ and _QualityProfileBackgroundProcessor_. I haven't added comprehensive tests to _ErrorListInfoBarControllerTests_ and _QualityProfileBackgroundProcessorTests_ - I've just added a single test to each to make sure they are passing the binding mode through correctly.